examples/cookbook/__mocks__/axios.ts

```diff
@@ -0,0 +1,13 @@
+const chuckNorrisError = () => {
```

Comment:
I don't think mocking axios is the right way to do this, imo you want to mock only api calls, not the whole library. There are librairies that are made for this such as msw, nock or shm which allow you to do that and also make tests fail if there are unmocked api calls, I think we should recommend those instead, wdyt?

Reply:
Appreciate the suggestion! I agree that mocking the entire axios library might be overkill, so I've revised the approach to override only the parts of the library that handle API calls—such as get, post, etc. This should ensure we're targeting just the API interactions while keeping the rest of the library intact. This approach also serves as a simple way to show developers how to guard themselves against unexpected API calls.

Reply:
Regarding the libraries you mentioned (msw, nock, shm), I appreciate the recommendation. For more context, I've addressed this in a similar way to a previous comment [here](https://github.com/callstack/react-native-testing-library/pull/1655#discussion_r1738028031). I aimed to keep the mocking approach basic and straightforward for beginners, but I’ve also updated the documentation to include a "Further Reading and Alternatives" section. This provides additional options for more advanced mocking scenarios, which should be helpful for developers looking for different tools.

Next PR I'll raise will touch MSW specifically, I've mentioned it in the PR's description 👍🏻